tests/baselines/reference/quickInfoDisplayPartsArrowFunctionExpression.baseline

```diff
@@ -73,7 +73,8 @@
           "kind": "keyword"
         }
       ],
-      "documentation": []
+      "documentation": [],
+      "tags": []
```

Comment:
And now look at those billions of empty arrays peppering the ***.baseline down from here.